### PR TITLE
Lightbox search

### PR DESCRIPTION
**Pull request by sroy3**

### CODE REVIEW

File: scss/components/lightbox-search.scss [hunk 1]
```diff
@@ -0,0 +1,58 @@
+.lightbox-search__box {
```
Comment (JPLachance) [inline scss/components/lightbox-search.scss]: It's the __ notation mandatory? I would suggest:
```
.lightbox-search {
  .box {...}
  .input{...}
}
```

File: scss/components/lightbox-search.scss [hunk 2]
```diff
@@ -0,0 +1,58 @@
...
+  outline: none;
+}
+
+&.admin #admin .admin-autocomplete .admin-popover-container .admin-popover {
+  width: 100%;
+  padding: 0;
+  margin-top: 5px;
+  overflow: hidden;
+
+  background: #ffffff;
```
Comment (JPLachance) [inline scss/components/lightbox-search.scss]: Use the @pure_white variable.

File: scss/components/lightbox-search.scss [hunk 3]
```diff
@@ -0,0 +1,58 @@
+.lightbox-search__box {
+  position: absolute;
+  left: 50%;
+
+  width: 90%;
+  max-width: 600px;
+
+  -webkit-transform: translateX(-50%);
```
Comment (JPLachance) [inline scss/components/lightbox-search.scss]: No need for that since we are using autoprefixer :)

File: scss/components/lightbox-search.scss [hunk 4]
```diff
@@ -0,0 +1,58 @@
...
+  padding: 10px 20px;
+
+  color: $block-text-color;
+  font-size: 2rem;
+
+  border: none;
+  outline: none;
+}
+
+&.admin #admin .admin-autocomplete .admin-popover-container .admin-popover {
```
Comment (JPLachance) [inline scss/components/lightbox-search.scss]: I'm not sure it's a good idea to put admin specific code into the Vapor styleguide.

File: scss/components/lightbox-search.scss [hunk 5]
```diff
@@ -0,0 +1,58 @@
...
+
+    color: $block-text-color;
+    line-height: inherit;
+
+    cursor: pointer;
+
+    &:hover, &.admin-active {
+      color: $medium-blue;
+
+      background-color: $table-editable-cell-color;
```
Comment (JPLachance) [inline scss/components/lightbox-search.scss]: We should not use that variable anymore. Use palette.scss colors.

File: scss/components/lightbox-search.scss [hunk 6]
```diff
@@ -0,0 +1,58 @@
...
+  }
+}
+
+.autocomplete-value {
+  display: block;
+  padding: 10px 20px;
+
+  font-size: 1.8rem;
+
+  border-top: 1px solid #eeeeee;
```
Comment (JPLachance) [inline scss/components/lightbox-search.scss]: Variable for the color.

File: scss/components/lightbox-search.scss (hunk 1 above)
Comment (wfortin) [inline scss/components/lightbox-search.scss]: +1

File: scss/components/lightbox-search.scss [hunk 7]
```diff
@@ -0,0 +1,58 @@
...
+  background: #ffffff;
+  border: none;
+  border-radius: 0;
+  border-radius: 3px;
+
+  ul li {
+    height: auto;
+    padding: 0;
+
+    color: $block-text-color;
```
Comment (JPLachance) [inline scss/components/lightbox-search.scss]: Same comment, use palette colors.

**Comment by wfortin** [comment]
Not much to add to what JP already pointed out! Great job! Can't wait to use that!

**Comment by JPLachance** [comment]
Great job! My only drawback is that it's really an admin specific piece of code. I'm not sure we should put in into Vapor.

File: scss/components/lightbox-search.scss [hunk 8]
```diff
@@ -0,0 +1,47 @@
...
+.lightbox-search .popover-container .lightbox-search-popover {
+  width: 100%;
+  padding: 0;
+  margin-top: 5px;
+  overflow: hidden;
+
+  background: $pure-white;
+  border: none;
+  border-radius: 0;
+  border-radius: 3px;
```
Comment (GermainBergeron) [inline scss/components/lightbox-search.scss]: border-radius seems duplicated

File: scss/components/lightbox-search.scss [hunk 9]
```diff
@@ -0,0 +1,46 @@
...
+
+.lightbox-search .popover-container .lightbox-search-popover {
+  width: 100%;
+  padding: 0;
+  margin-top: 5px;
+  overflow: hidden;
+
+  background: $pure-white;
+  border: none;
+  border-radius: 3px;
```
Comment (pochretien) [inline scss/components/lightbox-search.scss]: we use 2px for border radius , i think we have a variable for that

**Pull request merged**